spec/functions/fqdn_rand_string_spec.rb

```diff
@@ -57,7 +57,7 @@ def fqdn_rand_string(max, args = {})
 
     # workaround not being able to use let(:facts) because some tests need
     # multiple different hostnames in one context
-    allow(scope).to receive(:lookupvar).with('::fqdn', {}).and_return(host)
+    allow(scope).to receive(:lookupvar).with('facts', {}).and_return( { 'networking' => { 'fqdn' => host } })
```

Comment:
```suggestion
    allow(scope).to receive(:lookupvar).with('facts', {}).and_return({ 'networking' => { 'fqdn' => host } })
```

Reply:
Yeah my git is being difficult rn 😭  but I'll fix that once I get that figured out, thank you!